src/portal/src/i18n/lang/fr-fr-lang.json

```diff
@@ -130,6 +134,8 @@
         "CLI_PASSWORD_TIP": "The CLI secret can be used as a password, for Docker or Helm client. With the OIDC auth mode enabled, we strongly recommend using robot accounts, as CLI secrets depend on the validity of the ID token and require the user to regularly log in to the UI to refresh the token.",
```

Comment:
Could you translate this old one too please ? (not required)

Reply:
I missed that one. I will add it.